config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -5115,7 +5121,7 @@ rubrics.instance.layouts[flex].language[en-US].default-rating.title=New Rating
 rubrics.instance.layouts[flex].language[en-US].default-rating.description=New rating description
 rubrics.instance.layouts[flex].language[en-US].default-rating.points=0
 
-rubrics.instance.layouts[flex].language[es-ES].default-rubric.title=Nueva R\u00FAbrica
+rubrics.instance.layouts[flex].language[es-ES].default-rubric.title=Nueva R\u00fabrica
```

Comment:
why are you changing this line?

Reply:
Those other two changes happened during a rebase; I'll remove them.